### ISSUE TITLE
PROBLEM IN AUTOMATICALLY RUNNING A MOVIE FROM THE ANDROID TV HOME IN SMART PLAYLIST

### ISSUE THREAD

**Issue by boxcontrol1998** [xbmc/xbmc#21502]
How can i start a movie automatically (without pressing play in the movie tab) on the smart playlists in the android tv home?

within kodi by pressing on a movie, the movie starts automatically, but outside kodi in the android tv home, if I press on a movie (present in the smart playlists created by kodi and visible in the home) kodi opens but me opens the movie in an info tab, and I have to press play.

is it possible, instead of opening the info tab, that the film started immediately?

summing up:
within kodi the film starts immediately without problems.
outside of kodi (therefore in the home of android tv) in a playlist (created with kodi), if I press on a movie, kodi starts and then goes to the info tab of the movie. (without the possibility of starting it directly).

anyone with an android tv can try it.

is it possible to do this somehow also by editing an .xml file?

Thanks to those who will help me

**Comment by ksooo** [comment]
IS YOUR CAPS LOCK KEY BROKEN OR WHY DO YOU SHOUT IN THE ISSUE SUBJECT?
